android/sdk/src/main/java/org/jitsi/meet/sdk/RNConnectionService.java

```diff
@@ -226,4 +237,42 @@ void onCallAudioStateChange(android.telecom.CallAudioState callAudioState) {
     interface CallAudioStateListener {
         void onCallAudioStateChange(android.telecom.CallAudioState callAudioState);
     }
+
+    /**
+     * Helper function to send an event to JavaScript.
+     *
+     * @param eventName {@code String} containing the event name.
+     * @param data {@code Object} optional ancillary data for the event.
+     */
+    public void emitEvent(
+        String eventName,
+        @Nullable Object data) {
+        ReactContext reactContext = getReactApplicationContext();
+
+        if (reactContext != null) {
+            reactContext
+                .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class)
+                .emit(eventName, data);
+        }
+    }
+
+    /**
+     * Finds a native React module for given class.
+     *
+     * @param nativeModuleClass the native module's class for which an instance
+     * is to be retrieved from the {@link #(RNConnectionService)}.
+     * @return {@link NativeModule} instance for given interface type or
+     * {@code null} if no instance for this interface is available, or if
+     * {@link #(RNConnectionService)} has not been initialized yet.
+     */
+    public RNConnectionService getNativeModule(
```

Comment:
This is no longer necessary, is it?

Reply:
We use it in ConnectionService for the onCallAudioStateChanged method